docs/runbooks/CDIStorageProfilesIncomplete.md

```diff
@@ -0,0 +1,61 @@
+# CDIStorageProfilesIncomplete
+
+## Meaning
+
+An incomplete StorageProfile means that CDI will not be able to automatically infer PVC fields such as volumeMode, accessMode for the user's disk request.  
+
+This alert fires when any of the StorageProfiles are incomplete, and thus not inferrable.
+
+## Impact
+
+DataVolume's creation operation (build a VM disk on PVC) not succeeding.
+
+## Diagnosis
+
+- Find the storage profile that couldn't be fully populated by CDI using your desired storage class name:
+	```bash
+	kubectl get storageprofile <your_storage_class_name>
+	```
+
+- Follow the instructions in the 'Mitigation' section to populate the missing info.
+
+## Mitigation
+
+Please refer to the StorageProfile documentation, which states how one can provide the needed information in the StorageProfile spec section:  
+[Empty profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md#empty-storage-profile)  
+[User defined profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md#user-defined-storage-profile)
+
+An example of providing the missing info:
```

Comment:
could you please add some information about the meaning of these three things like

1. before
2. how to add
3. after

Reply:
Done